src/main/java/com/mraof/minestuck/data/MSBlockStateProvider.java

```diff
@@ -261,15 +387,32 @@ protected void registerStatesAndModels()
 		simpleBlockWithItem(MSBlocks.CRACKED_BLACK_STONE_BRICKS);
 		
 		simpleBlockWithItem(MSBlocks.FLOWERY_MOSSY_COBBLESTONE);
+		stairsWithItem(MSBlocks.FLOWERY_MOSSY_COBBLESTONE_STAIRS, MSBlocks.FLOWERY_MOSSY_COBBLESTONE);
+		slabWithItem(MSBlocks.FLOWERY_MOSSY_COBBLESTONE_SLAB, MSBlocks.FLOWERY_MOSSY_COBBLESTONE);
+		wallWithItem(MSBlocks.FLOWERY_MOSSY_COBBLESTONE_WALL, MSBlocks.FLOWERY_MOSSY_COBBLESTONE);
+		
+		//TODO update derivatives to use dynamic textures
 		variantsWithItem(MSBlocks.FLOWERY_MOSSY_STONE_BRICKS, 4,
 				i -> models().cubeAll("flowery_mossy_stone_bricks" + (i + 1),
 						texture(id("flowery_mossy_stone_bricks/" + (i + 1)))));
+		//stairsWithItem(MSBlocks.FLOWERY_MOSSY_STONE_BRICK_STAIRS, MSBlocks.FLOWERY_MOSSY_STONE_BRICKS);
+		//slabWithItem(MSBlocks.FLOWERY_MOSSY_STONE_BRICK_SLAB, MSBlocks.FLOWERY_MOSSY_STONE_BRICKS);
+		//wallWithItem(MSBlocks.FLOWERY_MOSSY_STONE_BRICK_WALL, MSBlocks.FLOWERY_MOSSY_STONE_BRICKS);
+		
 		variantsWithItem(MSBlocks.DECREPIT_STONE_BRICKS, 4,
 				i -> models().cubeAll("decrepit_stone_bricks" + (i + 1),
 						texture(id("decrepit_stone_bricks/" + (i + 1)))));
+		//stairsWithItem(MSBlocks.DECREPIT_STONE_BRICK_STAIRS, MSBlocks.DECREPIT_STONE_BRICKS);
+		//slabWithItem(MSBlocks.DECREPIT_STONE_BRICK_SLAB, MSBlocks.DECREPIT_STONE_BRICKS);
+		//wallWithItem(MSBlocks.DECREPIT_STONE_BRICK_WALL, MSBlocks.DECREPIT_STONE_BRICKS);
+		
 		variantsWithItem(MSBlocks.MOSSY_DECREPIT_STONE_BRICKS, 4,
 				i -> models().cubeAll("mossy_decrepit_stone_bricks" + (i + 1),
 						texture(id("mossy_decrepit_stone_bricks/" + (i + 1)))));
+		//stairsWithItem(MSBlocks.MOSSY_DECREPIT_STONE_BRICK_STAIRS, MSBlocks.MOSSY_DECREPIT_STONE_BRICKS);
+		//slabWithItem(MSBlocks.MOSSY_DECREPIT_STONE_BRICK_SLAB, MSBlocks.MOSSY_DECREPIT_STONE_BRICKS);
+		//wallWithItem(MSBlocks.MOSSY_DECREPIT_STONE_BRICK_WALL, MSBlocks.MOSSY_DECREPIT_STONE_BRICKS);
```

Comment:
This todo and commented code need resolving. The flowery mossy stone brick stairs/slab did have datagen before this PR, which might work as guidance.

Reply:
I intended to remove these and missed them. I can either
1. Remove them like I meant to, and work on them another time
2. Keep working at them now, at the expense of getting all the other blocks available for builders

Whatever you think is best!

Reply:
I'd lean towards getting them done now, but someone else can be the judge for how quickly this need to be merged and ready for builders.
But at the very least, the flowery mossy stone brick stairs/slab **should** be handled now since they were already present before this PR.

Reply:
Okay so when I look at FLOWERY_MOSSY_STONE_BRICK_STAIRS and FLOWERY_MOSSY_STONE_BRICK_SLAB I see that they're just using one texture (the first texture), i.e.:

`stairsWithItem(MSBlocks.FLOWERY_MOSSY_STONE_BRICK_STAIRS, "flowery_mossy_stone_brick", texture("flowery_mossy_stone_bricks/1"));`

So I updated slabs and walls for DECREPIT_STONE_BRICKS and MOSSY_DECREPIT_STONE_BRICKS to be consistent with that. I also modified the slab method to "match" the other two a little more, but I can revert this and go with the old way -- I just thought this looked a little more tidy.